### ISSUE TITLE
Hotstart Video isnt downloading!

### ISSUE THREAD

**Issue by Lucif3rHun1** [yt-dlp/yt-dlp#2669]
### Checklist

- [X] I'm reporting a broken site
- [X] I've verified that I'm running yt-dlp version **2022.02.04**. ([update instructions](https://github.com/yt-dlp/yt-dlp#update))
- [X] I've checked that all provided URLs are alive and playable in a browser
- [X] I've checked that all URLs and arguments with special characters are [properly quoted or escaped](https://github.com/ytdl-org/youtube-dl#video-url-contains-an-ampersand-and-im-getting-some-strange-output-1-2839-or-v-is-not-recognized-as-an-internal-or-external-command)
- [X] I've searched the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues including closed ones. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
- [X] I've read about [sharing account credentials](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#are-you-willing-to-share-account-details-if-needed) and I'm willing to share it if required

### Region

_India_

### Description

i am running it on my pc, Indian ip still facing issue.

### Verbose log

```shell
[debug] Command-line config: ['https://www.hotstar.com/in/movies/dhruva/1000192697/watch?utm_source=gwa', '-vU']
[debug] Encodings: locale cp1252, fs utf-8, out utf-8, err utf-8, pref cp1252
[debug] yt-dlp version 2022.02.04 [c1653e9] (win_exe)
[debug] Python version 3.8.10 (CPython 64bit) - Windows-10-10.0.22000-SP0
[debug] exe versions: ffmpeg 2022-02-03-git-e1a14479a8-full_build-www.gyan.dev (setts), ffprobe 2022-02-03-git-e1a14479a8-full_build-www.gyan.dev, phantomjs 2.1.1
[debug] Optional libraries: Cryptodome, mutagen, sqlite, websockets
[debug] Proxy map: {}
Latest version: 2022.02.04, Current version: 2022.02.04
yt-dlp is up to date (2022.02.04)
[debug] [hotstar] Extracting URL: https://www.hotstar.com/in/movies/dhruva/1000192697/watch?utm_source=gwa
[hotstar] 1000192697: Downloading JSON metadata
ERROR: [hotstar] 1000192697: Unable to download JSON metadata: HTTP Error 475: Unknown (caused by <HTTPError 475: 'Unknown'>); please report this issue on  https://github.com/yt-dlp/yt-dlp , filling out the "Broken site" issue template properly. Confirm you are on the latest version using -U (caused by <HTTPError 475: 'Unknown'>); please report this issue on  https://github.com/yt-dlp/yt-dlp , filling out the "Broken site" issue template properly. Confirm you are on the latest version using -U
  File "yt_dlp\extractor\common.py", line 730, in _request_webpage
  File "yt_dlp\YoutubeDL.py", line 3558, in urlopen
  File "urllib\request.py", line 531, in open
  File "urllib\request.py", line 640, in http_response
  File "urllib\request.py", line 569, in error
  File "urllib\request.py", line 502, in _call_chain
  File "urllib\request.py", line 649, in http_error_default
```

**Comment by Ashish0804** [comment]
```shell
[debug] Command-line config: ['-F', '--list-subs', '-v', 'https://www.hotstar.com/in/movies/dhruva/1000192697/watch?utm_source=gwa']
[debug] Encodings: locale cp1252, fs utf-8, out utf-8, err utf-8, pref cp1252
[debug] yt-dlp version 2022.02.04 [c1653e9ef]
[debug] Python version 3.8.8 (CPython 64bit) - Windows-10-10.0.19041-SP0
[debug] exe versions: ffmpeg N-105029-gf3a949dc4c-20211225 (setts), ffprobe N-105029-gf3a949dc4c-20211225
[debug] Optional libraries: Cryptodome, mutagen, sqlite, websockets
[debug] Proxy map: {}
[debug] [hotstar] Extracting URL: https://www.hotstar.com/in/movies/dhruva/1000192697/watch?utm_source=gwa
[hotstar] 1000192697: Downloading JSON metadata
[hotstar] 1000192697: Downloading webpage
[hotstar] 1000192697: Downloading token
[hotstar] 1000192697: Downloading JSON metadata
[hotstar] 1000192697: Downloading m3u8 information
[debug] Formats sorted by: hasvid, ie_pref, lang, quality, res, fps, hdr:12(7), vcodec:vp9.2(10), acodec, filesize, fs_approx, tbr, vbr, abr, asr, proto, vext, aext, hasaud, source, id
1000192697 has no subtitles
[info] Available formats for 1000192697:
ID           EXT RESOLUTION FPS │   FILESIZE   TBR PROTO  │ VCODEC        VBR ACODEC    ABR
────────────────────────────────────────────────────────────────────────────────────────────
sdr-hls-97   mp4 250x140     25 │ ~110.53MiB   97k m3u8_n │ avc1.42c015   97k mp4a.40.5  0k
sdr-hls-148  mp4 320x180     25 │ ~168.43MiB  148k m3u8_n │ avc1.42c015  148k mp4a.40.5  0k
sdr-hls-260  mp4 320x180     25 │ ~294.60MiB  260k m3u8_n │ avc1.42c015  260k mp4a.40.2  0k
sdr-hls-401  mp4 416x234     25 │ ~454.79MiB  401k m3u8_n │ avc1.42c01e  401k mp4a.40.2  0k
sdr-hls-409  mp4 640x360     25 │ ~463.17MiB  409k m3u8_n │ avc1.42c01e  409k mp4a.40.2  0k
sdr-hls-499  mp4 720x404     25 │ ~565.27MiB  499k m3u8_n │ avc1.42c01e  499k mp4a.40.2  0k
sdr-hls-909  mp4 1280x720    25 │ ~  1.01GiB  909k m3u8_n │ avc1.4d401f  909k mp4a.40.2  0k
sdr-hls-1320 mp4 1600x900    25 │ ~  1.46GiB 1320k m3u8_n │ avc1.4d4028 1320k mp4a.40.2  0k
sdr-hls-1861 mp4 1920x1080   25 │ ~  2.06GiB 1861k m3u8_n │ avc1.4d4028 1861k mp4a.40.2  0k
```

**Comment by pukkandan** [comment]
Confirm whether you are able to access the video in browser using the same IP
